clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/check_cluster_state.py

```diff
@@ -33,7 +34,24 @@ def describe_clusters():
     if sites != "" and etcd_version != "2.2.5":
         local_site_info = " in the local site (" + local_site + ")"
 
-    print "This script prints out the status of the Chronos, Memcached and Cassandra clusters{}.\n".format(local_site_info)
+    print "This script prints the status of the Chronos, Memcached and Cassandra clusters{}.".format(local_site_info)
+
+    plugin_dir = '/usr/share/clearwater/clearwater-cluster-manager/plugins'
+    if os.path.isdir(plugin_dir):
+        plugins = [plugin_name[:-10] for plugin_name in os.listdir(plugin_dir) if
+                plugin_name.endswith('_plugin.py')]
+        plugins.remove('memcached_remote')
+        
+        if len(plugins) >= 2:
+            cluster_str = "{} and {}".format(", ".join(plugins[:-1]),
+                    plugins[-1])
+        else:
+            cluster_str = str(plugins[0])
+
+        print "This node ({}) should be in {} cluster.\n".format(local_node_ip,
```

Comment:
Can the phrasing here be 'The node should be in the Chronos, Memcached and Cassandra clusters' - I've added 'the', pluralised clusters (but keep it as cluster if there's only one plugin), and capitialised the first letter of the cluster type.